Data structures/List items/task.md

```diff
@@ -7,6 +7,10 @@ This way you can even change the size of a list or clear it entirely by doing so
 animals[:] = []
 ```
   
+### Task
 Make all `animals` elephants by replacing the last two items.  
+We suggest that you first run the program in the IDE without changes to see how the `animals` list looks like after all the operations have been made
```

Comment:
We suggest that you first run the program in the IDE without making any changes. This will allow you to see what the `animals` list looks like after all the operations have been performed.